symphony-bdk-examples/bdk-app-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -24,6 +28,6 @@ server:
     key-store-password: password
     key-store-type: pkcs12
     key-alias: tomcat
-  port: 8443
+  port: 443
```

Comment:
`443` is a default port and I'm not sure that all developers can start a server on this port locally. Let's use `10443`